app/views/passengers/index.haml

```diff
@@ -14,6 +14,9 @@
               checked: @filter == filter,
               inline: true
       = f.submit 'Print'
+      -if @current_user.admin?
+        .btn.btn-secondary.pull-right#copybtn Copy Emails
```

Comment:
Seems like this button shouldn't be _within_ the "print" form.

Reply:
Also, `pull-right` seems a little weird.

Reply:
Okay, I guess the button for copying can be below the button for print. Screenshot
 attached:

![Screen Shot 2020-01-09 at 1 58 58 PM](https://user-images.githubusercontent.com/12656859/72096496-4a629e80-32e8-11ea-956b-8f3294b74d69.png)